.github/workflows/pipeline.yaml

```diff
@@ -35,12 +38,25 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_HUB_ACCESS_USER }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+      - name: Login to azure
+        uses: azure/login@v1
+        with:
+          creds: '${{ secrets.AZURE_CREDENTIALS }}'
+      - name: 'Run Azure CLI commands'
+        run: |
+          az account show
+          az group list
+          pwd 
       - name: Checkout files 
         uses: actions/checkout@v2
       - name: Build docker
         run: docker build --target production --tag $DOCKER_HUB_USER/todo-app:latest .
       - name: Push to docker hub
         run: docker push $DOCKER_HUB_USER/todo-app:latest
+      - name: Terraform init
+        run: terraform init
+      - name: Deploy app with terraform
+        run: terraform apply -auto-approve
```

Comment:
Currently this is failing - [see the latest job](https://github.com/MHolmes219/DevOps-Course-Starter/actions/runs/3298956623)

I can't see any evidence that you've switched to remote state (Part 3) since we're missing the relevant backend block which would look something like:
```terraform
    backend "azurerm" {
        resource_group_name  = "tfstate"
        storage_account_name = "<storage_account_name>"
        container_name       = "tfstate"
        key                  = "terraform.tfstate"
    }
```

Reply:
Resolved